agent/engine/errors.go

```diff
@@ -154,15 +159,19 @@ func (err CannotStopContainerError) ErrorName() string {
 	return "CannotStopContainerError"
 }
 
-func (err CannotStopContainerError) IsUnretriableError() bool {
+// When stopping a container, most errors that we can get should be
```

Comment:
You have this in a godoc location, but it's not godoc format.  Change the beginning to read "IsRetriableError returns ..."